src/devices/wholeBodyDynamics/WholeBodyDynamicsDevice.h

```diff
@@ -348,6 +393,19 @@ class WholeBodyDynamicsDevice :  public yarp::dev::DeviceDriver,
     /** F/T sensors interfaces */
     std::vector<yarp::dev::IAnalogSensor * > ftSensors;
 
+    /** Remapped multiple analog sensors containing the sensors that implement multiple analgo sensor interfaces*/
+    yarp::dev::PolyDriver multipleAnalogRemappedDevice;
+    struct
+    {
+        yarp::dev::ITemperatureSensors * temperatureSensors;
+        yarp::dev::ISixAxisForceTorqueSensors * ftMultiSensors;
+        yarp::dev::IMultipleWrapper * multwrap;
+    } remappedMASInterfaces;
+    yarp::sig::Vector ftTempMapping;
```

Comment:
If this is a int --> int mapping, can we just use a `std::vector<int>` or `std::vector<size_t>`?

Reply:
changed to std::vector<int>